docs/charts/bar.md

```diff
@@ -152,7 +152,7 @@ The `data` property of a dataset for a bar chart is specified as a an array of n
 data: [20, 10]
 ```
 
-You can also specify the dataset as x/y coordinates when using the [time scale](./time.md).
+You can also specify the dataset as x/y coordinates when using the [time scale](../axes/cartesian/time.md#Time Cartesian Axis).
```

Comment:
The link doesn't look correct, should be either `../axes/cartesian/time.md#time-cartesian-axis` or simply `/axes/cartesian/time.md`

Reply:
Done